sources/reference/resource-state.md

```diff
@@ -0,0 +1,38 @@
+page_main_title: state
+main_section: Reference
+sub_section: Resources
+
+# state
+A `state` resource type is used to create a central state that can be used across the entire pipelines. This resource can be used as an IN input and OUT output for [any job](jobs-overview/). The timezone used for triggering jobs is UTC.
+
+You can create this resource by adding it to `shippable.resources.yml`
+
+```
+resources:
+  - name: res-state
+    type: state
+```
+
+* `name` should be an easy to remember text string. This will appear in the visualization of this resource in the SPOG view. It is also used to refer to this resource in the `shippable.jobs.yml`. If you have spaces in your name, you'll need to surround the value with quotes, however, as a best practice we recommend not including spaces in your names.
+
+* `type` is always set to 'state'.
+
+You can use the time resource as an input and output for any job you want to trigger at that interval:
+
+```
+jobs:
+  - name: tfDeploy
+    type: runSh
+    steps:
+      - IN: integration-aws
+      - IN: repo-tfScripts
+      - IN: res-state
+      - TASK:
+        - script: ./doSomething.sh
```

Comment:
I think it would be better to show a couple script lines that actually write to the state resource. nothign elaborate, but maybe one that writes to its .env file, and one that writes and actual file to it's state directory. not a full blown example, but just a simple usage to help a user understand what any of this means.